xarray/plot/facetgrid.py

```diff
@@ -282,13 +282,37 @@ def __init__(
         self._mappables = []
         self._finalized = False
 
+    @property
+    def axes(self) -> np.ndarray:
+        warnings.warn(
+            (
+                "self.axes is deprecated since 2022.11 in order to align with "
```

Comment:
Maybe there will be a 2022.10.1, haha

Reply:
That's the anoying part of the new scheme...